set.mm

```diff
@@ -31823,10 +31802,9 @@ requires either a dv condition ( ~ sb56 ) or a non-freeness hypothesis
 
   ${
     $d w x $.  $d w y $.
-    $( Alternate version of ~ axc11nlemOLD2 used in an older proof.
-       (Contributed by NM, 8-Jul-2016.)  (Proof shortened by Wolf Lammen,
-       17-Feb-2018.)  (Proof modification is discouraged.)
-       (New usage is discouraged.) $)
+    $( Used in an older proof.  (Contributed by NM, 8-Jul-2016.)  (Proof
```

Comment:
I propose:
```
$( Lemma for ~ axc11nALT .  (Contributed ...
```

Reply:
axc11nlemALT is a weak replacement for aev in the proof of axc11n.  aev is provable from surprisingly few axioms.  Since axc11nlemALT is a specialization of aev, it is in principle not based on more axioms.  But its proof uses more, which is justified only, if it is structurally based on theorems you likely accept as more primitive, or that are more versatile, if you modify the axioms system slightly.  Whatever it is, it should somehow be expressed in axc11nALT, otherwise tirix question (shall we get rid of this?) will silently or openly crop up again.  That said, I will modify the text of axc11nlemALT as suggested on the next occasion.

Reply:
@wlammen @tirix : Indeed, after looking at it, I think both axc11nlemALT and axc11nALT can be removed (the proof of axc11nALT is basically the same as that of axc11n once the use of axc11nlemALT is replaced by aev).

Reply:
axc11nALT is not exactly the topic of this PR.  I routinely checked the file for OLD theorems, and, so to say, by accident stumbled over axc11nlemALT, because a theorem mentioned in the introducing text had disappeared.  I would like to leave it here as is and suggest to open a different pull request, where the removal of axc11n*ALT is addressed.  I think that would be the right place to discuss this in greater detail.

Reply:
Ok. Current PR can be merged now.